packages/components/menu/src/menu.stories.ts

```diff
@@ -89,8 +128,7 @@ export const Grouped: Story = {
         <sl-icon name="far-gear"></sl-icon>
         Settings
       </sl-menu-item>
-      <hr />
-      <sl-menu-item-group heading="Design System">
```

Comment:
In figma we have examples with group title/heading:

<img width="600" alt="image" src="https://github.com/user-attachments/assets/bf6d6503-b560-469f-a99d-c13e3956fbff">


Can you please also add an example with group heading?